tests/README.md

```diff
@@ -27,14 +27,21 @@ compares it (string compare) with the expected result, a file named by the
 second argument. The `NAME` macro holds the test name,  which is `call_avg` in
 this case.
 
+These tests run as part of the normal suite of unit tests if you are running LLVM 12.
+If not, you need to install 'nix' and run these tests via this script:
+`./tests/codegen-tests.sh`.
+
 #### Updating
 
-Run `./scripts/update_codegen_tests.sh` after making codegen changes up update
-the expected LLVM IR.
+**LLVM 12**
+
+If you are running LLVM 12 or want to only update specifc tests with `--gtest_filter`
```

Comment:
Typo:
```suggestion
If you are running LLVM 12 or want to only update specific tests with `--gtest_filter`
```

Reply:
Good catch. I gotta run the spell checker more often.